libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+          Name: 'Mayor', // Contest name, not for ballot
+          BallotTitle: {
+            // Display name for contest on ballot
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Mayor',
+              },
+            ],
+          },
+          VotesAllowed: 1,
+          // TODO there's no field analogous to "allowWriteIns" in CDF
+          // Contest options reference the candidates defined above by id
+          ContestOption: [
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['sherlock-holmes'],
+            },
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['thomas-edison'],
+            },
+          ],
+        },
+      ],
+
+      // Definition of ballot styles
+      BallotStyle: [
+        {
+          '@type': 'BallotDefinition.BallotStyle',
+          '@id': 'ballot-style-1',
+          // TODO do we also reference districts here? what's the relationship between districts and precincts?
+          GpUnit: ['precinct-23', 'precinct-22'],
+        },
+      ],
+    },
+  ],
+
+  // Definition of parties
+  Party: [
+    {
+      '@type': 'BallotDefinition.Party',
+      '@id': 'party-0',
+      // TODO there's no way to distinguish between "name" and "fullName"
+      // This Name goes on the ballot
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Democrat',
+          },
+        ],
+      },
+      Abbreviation: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'D',
+          },
+        ],
+      },
+    },
+  ],
+
+  // Definition of "geo-political units" (e.g. states, counties, districts, precincts, etc)
+  GpUnit: [
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'state-hamilton',
+      Type: 'state',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'State of Hamilton',
+          },
+        ],
+      },
+      ComposingGpUnit: ['county-franklin'],
+    },
+    {
+      '@type': 'BallotDefinition.ReportingUnit',
+      '@id': 'county-franklin',
+      Type: 'county',
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Franklin County',
+          },
+        ],
+      },
+      // TODO what's the relationship between counties and districts?
```

Comment:
GpUnits in CDF are flexible and can model many different hierarchies. VX election def seems more constrained. How should we approach that?

Reply:
VVSG requires that we support up to three levels of geographic hierarchy and we only have the hardcoded two. The desired levels are not going to be consistent across customers and elections, which is a problem we've already run into in NH and MS. I think we're going to need to move to a flexible model where we aren't constrained to the district/precinct hierarchy only.